history/countries/SYR - Syria.txt

```diff
@@ -87,12 +110,12 @@ create_country_leader = {
 	ideology = marxism
 }
 
-create_country_leader = {
-	name = "Michel Aflaq"
-	desc = ""
-	picture = "gfx/leaders/SYR/r56_portrait_SYR_Aflaq.dds"
-	ideology = arab_socialism
-}
+#create_country_leader = {
```

Comment:
Add a reason why you're commenting him out, since he doesn't seem to be a character, he even has a portrait